packages/typescript/BUILD.bazel

```diff
@@ -38,13 +45,6 @@ genrule(
     cmd = "cp $< $@",
 )
 
-genrule(
```

Comment:
why remove this? I'd rather have fewer INTERNAL fences, they are harder to reason about

Reply:
Needed
```
js_library(
    name = "protobufjs_bootstrap_scripts",
```
in BUILD.bazel

Reply:
for the protobuf example

Reply:
okay but I don't think the fencing is required here?

Reply:
good point. removed.